tox.ini

```diff
@@ -17,3 +17,7 @@ changedir = {toxinidir}/cfgov
 commands=
     python manage.py test {posargs}
     
+[testenv:fast]
```

Comment:
There are two different and valuable use cases. First, start a brand new venv, create a database with all migrations, and then run the set of unit tests. This takes a long time but makes sure that everything works from scratch. Second, is reuse the existing venv (avoids having to pip install each time) and avoid using the DB at all (for now, although we should discuss how to handle this going forward). This is fast and makes it easy for developers to run tests quickly.

Travis and Jenkins should do the former, and I think it's useful to have the latter for day-to-day work. My instinct is that it's best to leave the default be the former when running `tox`, and explicitly use the faster one with `tox -e fast`, but we could change it so that the default is fast and you have to do something like `tox -e rebuild` to do everything.